specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -130,7 +129,7 @@ namespace Azure.Messaging.EventGrid {
   }
 
   @doc("Lock token input formatting.")
-  model LockTokenInput {
+  model LockTokens {
```

Comment:
Thinking about this more, I'm not sure that I agree with the change. I think the reason we have a model here to begin with is that we expect that this may need to evolve over time (to be frank, this seems dubious), but given that, we should probably name the model with this in mind. LockTokenInput isn't great either, and it would be great to avoid needing separate models for both Acknowledge and Release, but in the spirit of evolvability, perhaps that would be the lesser of the two evils, i.e. having separate AcknowledgeInput and ReleaseInput models? This would need to be run by the service folks as well.